src/d3d11/d3d11_context.cpp

```diff
@@ -662,9 +662,13 @@ namespace dxvk {
     if (!buf || !uav)
       return;
 
+    auto counterSlice = uav->GetCounterSlice();
+    if (!counterSlice.defined())
+      return;
+
     EmitCs([
       cDstSlice = buf->GetBufferSlice(DstAlignedByteOffset),
-      cSrcSlice = uav->GetCounterSlice()
+      cSrcSlice = counterSlice
```

Comment:
Can we get an `std::move` here? Saves a few CPU cycles due to atomic ref counts not getting changed redundantly, although to be fair, this function is hardly performance-critical.

Reply:
Yes, done.